examples/text-sentiment/start_runtime.py

```diff
@@ -0,0 +1,12 @@
+from os import path
+import sys
+import alog
+
+sys.path.append(path.abspath(path.join(path.dirname(__file__), "../"))) # Here we assume this file is at the same level of requirements.txt
```

Comment:
sorry this was my change from earlier - I think this would make more sense
```suggestion
sys.path.append(path.abspath(path.join(path.dirname(__file__), "../"))) # Here we assume this `start_runtime` file is at the same level of the `text_sentiment` library
```

Reply:
Updated